spec/index.bs

```diff
@@ -10804,6 +10805,12 @@ It must only be included by interfaces which also include those mixins.
     : <dfn>\[[dynamic_offsets]]</dfn>, of type [=ordered map=]&lt;{{GPUIndex32}}, [=list=]&lt;{{GPUBufferDynamicOffset}}&gt;&gt;, initally empty
     ::
         The current dynamic offsets for each {{GPUBindingCommandsMixin/[[bind_groups]]}} entry.
+
```

Comment:
Maybe we could replace `[[bind_groups]]` with `[[bind_groups_layouts]]` or track both? This could avoid the need for placeholder bind groups as a concept. WDYT?

Reply:
I wondered about that myself. I'll take a pass at it and see how complicated it would be.